GUI/MainWindow.xaml.cs

```diff
@@ -1409,5 +1412,26 @@ private void AddContaminantXML_Click(object sender, RoutedEventArgs e)
             }
             dataGridProteinDatabases.Items.Refresh();
         }
+
+        // handle window closing
+        private void MainWindow_Closing(object sender, CancelEventArgs e)
+        {
+            if (!GuiGlobalParams.DisableCloseWindow)
```

Comment:
`if (!GuiGlobalParams.DisableCloseWindow && !GlobalVariables.Contains("DEBUG"))`

just makes it less annoying for us if you're working on a debug version and your preferences can't be saved